nixos/tests/yabar.nix

```
@@ -1,12 +1,8 @@
 import ./make-test-python.nix ({ pkgs, lib, ... }:
 
-with lib;
-
 {
   name = "yabar";
-  meta = with pkgs.lib.maintainers; {
-    maintainers = [ ];
-  };
+  meta.maintainers = [ ];
```

```suggestion
  meta.maintainers = with lib.maintainers; [ ];
```

Yes, this is technically unused but it is often kept to show that there is no maintainer. It would be fine for me to remove those to make it easier for tooling to handle this.

Also pkgs.lib should not be used when we can easily get a lib input.